lib/install.go

```diff
@@ -65,7 +65,7 @@ func install(product Product, tfversion, binPath, installPath, mirrorURL, goarch
 	exist := versionExist(tfversion, tflist) // Check if version exists before downloading it
 
 	if !exist {
-		return fmt.Errorf("the provided %s version does not exist: %q.\n Try `tfswitch -l` to see all available versions", product.GetId(), tfversion)
+		return fmt.Errorf("Provided %s version does not exist: %q.\n\tTry `tfswitch -l` to see all available versions", product.GetId(), tfversion)
```

Comment:
I think this would read better as "provided version %s does not exist" ?

Reply:
Yeah, good catch. Changed to `Requested version %s does not exist` via f03eb52 via https://github.com/warrensbox/terraform-switcher/pull/549

Reply:
hahaha, just noticed that `%s` in this place is for product name =))) reverted in #549